person.rb

```diff
@@ -25,6 +26,15 @@ def correct_name
     @name
   end
 
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
+
```

Comment:
- [ ] Well done working on the requested changes 👏. However, as pointed out by the previous reviewer, this method should be added to the `Student` class and not to the `Person` class. Please ensure to move the method to the `Student` class.  :ok_hand: 😊